.azure/gpu-tests-lite.yml

```diff
@@ -41,12 +41,12 @@ jobs:
     timeoutInMinutes: "20"
     # how much time to give 'run always even if cancelled tasks' before stopping them
     cancelTimeoutInMinutes: "2"
-    pool: azure-jirka-spot
+    pool: lit-rtx-3090
     container:
       image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1"
       # default shm size is 64m. Increase it to avoid:
       # 'Error while creating shared memory: unhandled system error, NCCL version 2.7.8'
-      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=512m"
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=${Agent.Name##*_} --shm-size=2gb"
```

Comment:
What does this VISIBLE_DEVICES change do?

Reply:
I am trying to have up to 4 agents per machine where each is using a different GPU, so:
- agent 0: `CUDA_VISIBLE_DEVICES=0,1`
- agent 1: `CUDA_VISIBLE_DEVICES=2,3`
- agent 2: `CUDA_VISIBLE_DEVICES=4,5`
- agent 3: `CUDA_VISIBLE_DEVICES=6,7`